src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -24,7 +24,9 @@ public sealed class BinaryLogger : ILogger
         // version 4:
         //   - new TargetSkippedEventArgs
         //   - new TargetStartedEventArgs.BuildReason
-        internal const int FileFormatVersion = 4;
+        // version 5:
+        //   - new EvaluationFinished.ProfilerResult
+        internal const int FileFormatVersion = 5;
```

Comment:
>= 5; [](start = 45, length = 4)

quick question. I have another small PR in progress that modifies the evaluation location object to add specific tracking for globbing.
This actually changes the structure of the location and turns the 'is element'  field into an enum.
Not a big deal, but we will need to bump the version of the binary logger again and do the appropriate checks.
Maybe we want this other PR to be pushed first so we don't need to do this?

Reply:
I'd rather we didn't have an extra version. @smera, can you rebase your work on this PR so you can reuse the version?

---
In reply to: [156224871](https://github.com/Microsoft/msbuild/pull/2788#discussion_r156224871) [](ancestors = 156224871)

Reply:
Sure, as soon as it is ready to be pushed I'll rebase on top of this one, and then I can push both together

---
In reply to: [156255138](https://github.com/Microsoft/msbuild/pull/2788#discussion_r156255138) [](ancestors = 156255138,156224871)